packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json.test.ts

```diff
@@ -451,45 +435,9 @@ test(testCase, async (t: Test) => {
       '{type="' +
       K_CACTUS_QUORUM_TOTAL_TX_COUNT +
       '"} 5';
-    t2.ok(res);
-    t2.ok(res.data);
-    t2.equal(res.status, 200);
-    t2.true(
-      res.data.includes(promMetricsOutput),
-      "Total Transaction Count of 5 recorded as expected. RESULT OK.",
-    );
-    t2.end();
+    expect(res);
+    expect(res.data);
+    expect(res.status).toEqual(200);
+    expect(res.data).toContain(promMetricsOutput);
   });
-
-  test("deploys contract via .json file with constructorArgs", async (t2: Test) => {
```

Comment:
@awadhana Maybe a rebase gone wrong here, this test seems to be missing from the Jestified version.

Reply:
Yup, I'm going to have to re-review this one and add that one back. Thank you, I missed that!

Reply:
@awadhana No worries, similar things happen to me all the time too!